### PR TITLE
fix(SelectValueObserver) Safari and Internet Explorer

### PR DESCRIPTION
**Pull request by jdanyow**
Removed the `HTMLSelectElement.selectedItems` polyfill because it caused problems with Safari and only Internet Explorer needed it.  All tests passing and the integration tests work in Chrome, Firefox and IE.  Wasn't able to test Safari because of another issue is preventing the app from loading.  Here's the safari error at startup:

``` bash
[Error] SyntaxError: Cannot declare a parameter named 'fetch' as it shadows the name of a strict mode function.
    appendChild (skeleton-navigation, line 88)
    doEval (system.js, line 1)
    __eval (system.js, line 1)
    a (system.js, line 1)
    instantiate (system.js, line 1)
    instantiate (system.js, line 1)
    instantiate (system.js, line 1)
    instantiate (system.js, line 1)
    instantiate (system.js, line 1)
    instantiate (system.js, line 1)
    (anonymous function) (es6-module-loader.js, line 7)
    O (es6-module-loader.js, line 7)
    K (es6-module-loader.js, line 7)
    when (es6-module-loader.js, line 7)
    run (es6-module-loader.js, line 7)
    _drain (es6-module-loader.js, line 7)
    drain (es6-module-loader.js, line 7)
[Error] Potentially unhandled rejection [3] Error loading "github:aurelia/loader-default@0.4.3/index" at http://jdanyow.github.io/skeleton-navigation/jspm_packages/github/aurelia/loader-default@0.4.3/index.js
Error loading "github:aurelia/loader-default@0.4.3/index" from "github:aurelia/loader-default@0.4.3" at http://jdanyow.github.io/skeleton-navigation/jspm_packages/github/aurelia/loader-default@0.4.3.js
Error evaluating http://jdanyow.github.io/skeleton-navigation/jspm_packages/github/aurelia/loader-default@0.4.3/index.js
SyntaxError: Cannot declare a parameter named 'fetch' as it shadows the name of a strict mode function. (WARNING: non-Error used)
    (anonymous function) (es6-module-loader.js, line 7)
    f (es6-module-loader.js, line 7)
    i (es6-module-loader.js, line 7)
```

### CODE REVIEW

**Comment by EisenbergEffect** [comment]
I'm confused a bit. This fixes Safari...but does it break IE again?

**Comment by EisenbergEffect** [comment]
Is it just an alternate way of implementing the same thing?

**Comment by jdanyow** [comment]
Sorry for the thrash on this item- here's what happened:

1. Initial implementation used HTMLSelectElement.selectedOptions.  Worked wonderfully in chrome
2. Got around to testing with Internet Explorer, which doesn't support HTMLSelectElement.selectedOptions... wrote a polyfill.  Tested in chrome, IE, firefox.  All tests passing, etc.
3. Posted url on the main gitter... someone reported it didn't work in safari.  The error message they reported said something about "selectedOptions" being deprecated (doesn't say this on MDN).  
4. Did what I should have done in the first place, refactored the code and to remove all uses of "selectedOptions", in favor of scanning the "options" property and checking each option's "selected" flag.  Tested in chrome, firefox and IE- all good.  @PWKad tested in safari and hit the unrelated? error above before the app finished bootstrapping.
5. Submitted this PR.

tldr; removed uses of `selectedOptions`, works in Chrome/Firefox/IE, unable to test in Safari because of unrelated error bootstrapping

**Comment by EisenbergEffect** [comment]
That bug looks like it's coming from inside of the es6 module loader or system.js more likely. Either way, I will get this merged. Thanks for trudging through this!

**Comment by EisenbergEffect** [comment]
Just tested in Safari. Looks like the error is coming from the new loader plugin. The ES6 loader api defines a callback named `fetch`. I'll see if there's any way I can get around this.
